src/spikeinterface/core/core_tools.py

```diff
@@ -476,3 +477,24 @@ def normal_pdf(x, mu: float = 0.0, sigma: float = 1.0):
     """
 
     return 1 / (sigma * np.sqrt(2 * np.pi)) * np.exp(-((x - mu) ** 2) / (2 * sigma**2))
+
+
+def get_class_info(a_class):
+    """
```

Comment:
I think this docstring is doing too little.
```suggestion
    """
    Retrieve the import provenance of a class, including its an import name that consists of the class name and the module, the top-level module, and the module version.

    Parameters
    ----------
    a_class : type
        The class object for which to retrieve the import provenance.

    Returns
    -------
    dict
        A dictionary containing:
        - 'class':  The module path and the name of the class concatenated (e.g., 'package.subpackage.ClassName').
        - 'module': The top-level module name where the class is defined.
        - 'version': The version of the module if available, otherwise 'unknown'.
```